scripts/copydocs.py

```diff
@@ -69,7 +69,7 @@ def parse_args():
     parser.add_option("-c", "--comment", dest="comment", default="", help="comment")
     parser.add_option("--src", dest="src", metavar="SOURCE_URL", default="http://openlibrary.org/", help="URL of the source server (default: %default)")
     parser.add_option("--dest", dest="dest", metavar="DEST_URL", default="http://localhost", help="URL of the destination server (default: %default)")
-    parser.add_option("-r", "--recursive", dest="recursive", action='store_true', default=False, help="Recursively fetch all the referred docs.")
+    parser.add_option("-r", "--recursive", dest="recursive", action='store_true', default=True, help="Recursively fetch all the referred docs.")
```

Comment:
@cclauss Let's leave the default alone to avoid unintended consequences (and keep this PR to a bugfix :) ). We'd have to do update some of the Wiki pages if we change it. (Also we'd have to update this to `store_false`, wouldn't we?)

Reply:
Just to be clear, without this change to the default behavior, most copies seem to fail because a referenced entity is missing in the local environment.  This seems like new behavior compared to a month or so ago, IIRC, and I don't know why it happens.

E.g., works entity from a book isn't present:
```
$ ./scripts/copydocs.py /books/OL1810541M 2>&1 | egrep "400|books|works"
fetching ['/books/OL1810541M']
saving ['/books/OL1810541M']
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost/api/save_many
openlibrary.api.OLError: 400 Client Error: Bad Request for url: http://localhost/api/save_many. Response: {"error": "notfound", "key": "/works/OL19003703W", "at": {"key": "/books/OL1810541M", "property": "works"}}
```

Or authors entity from a work isn't present:
```
$ ./scripts/copydocs.py /works/OL19003703W 2>&1 | egrep "400|books|works"
fetching ['/works/OL19003703W']
saving ['/works/OL19003703W']
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost/api/save_many
openlibrary.api.OLError: 400 Client Error: Bad Request for url: http://localhost/api/save_many. Response: {"error": "notfound", "key": "/authors/OL3343129A", "at": {"key": "/works/OL19003703W", "property": "authors.author"}}
```

I thought changing the default would be a low-risk way to quickly unblock people, and if we think it's worth more investigation, we can do that less urgently.  If we proceed, I guess we'd want to add a ```--no-recursive``` flag that has ```dest='recursive'``` and ```action='store_false'``` to allow people to turn it off, even through the copy rarely works when it's disabled in my testing.

Thoughts?